content/spin/rust-components.md

```diff
@@ -102,6 +102,8 @@ fn hello_world(_req: Request) -> Result<Response> {
 }
 ```
 
+In the previous snippet, we used `http::Request::builder()`, which is provided by the `http` crate. Ensure that `http` is part of your dependencies in `Cargo.toml`. You must manually add the dependency for projects created using templates other than `http-rust`.
```

Comment:
Suggestion: it might be good to indent this paragraph as a "side note", similar to the way the hosts remark is indented at line 147, so as to avoid breaking the flow for people who are using the template (which we currently expect to be the most common case).